### DISCUSSION TITLE
CMIInventoryEditEvent for the API

### DISCUSSION THREAD

**Discussion by LogGits** [Zrips/CMI#4184] [Feature Requests]
**Description of feature/improvement request:** An event that gets called and if cancelled doesnt run player.updateInventory() for `/cmi inv`.

---
### Detailed explanation of what it should do, when and in what situation: ###
The event should run when a staff member goes edits an inventory using `/cmi inv` whereby if cancelled doesnt run the player.updateInventory().
### Basic explanation as to why this feature is needed and/or useful to have: ###
I need this so I staff can edit players inventories without triggering updates while the player is afk as i have a packet gui open during afk.

### Intended use-case: ###
```java
@EventHandler
public void onInventoryEdit(CMIInventoryEditEvent event){
    event.setCancelled(event.getPlayer.isAfk());
}
```

**Comment by CloudeLecaw** [comment]
Why it important that player is not afk if you edit his inventory?
